README.md

```diff
@@ -200,6 +200,33 @@ Instances of these can be started using the `docker-compose.yml` file.
 
 ### Outside of Docker
 
+#### Prerequisites
+
+##### MongoDB
+
+You must have access to a MongoDB instance with a database and be able to create the following indexes using either
+MongoDB Compass or a CLI e.g.
+
+```bash
+mongosh DATABASE_NAME --username USERNAME --password PASSWORD --authenticationDatabase=admin \
+   --eval 'db.attachments.createIndex({ "entity_id": 1, "code": 1 }, { name: "attachments_file_name_uniqueness_index", unique: true })' \
+   --eval 'db.images.createIndex({ "entity_id": 1, "code": 1 }, { name: "images_file_name_uniqueness_index", unique: true })'
+```
```

Comment:
Will these commands work if the collections have not been created? They only get created when the application starts and it adds something to the db.

Reply:
I tested it by dumping my database (and for good measure `sudo rm -rf mongodb/data/`), then running `docker compose up`, and it just creates the empty collections with the index. So it should work with or without any existing data (just as long as it doesn't already have the index)